{
	"schema_version": "1.6.0",
	"id": "CVE-2020-28941",
	"modified": "2024-11-21T05:23:20Z",
	"published": "2020-11-19T19:15:11Z",
	"related": [
		"DLA-2483-1",
		"SUSE-SU-2020:3713-1",
		"SUSE-SU-2020:3748-1",
		"SUSE-SU-2020:3764-1",
		"UBUNTU-CVE-2020-28941",
		"USN-4749-1",
		"USN-4750-1",
		"USN-4751-1",
		"openSUSE-SU-2020:2161-1",
		"openSUSE-SU-2020:2260-1",
		"openSUSE-SU-2021:0242-1"
	],
	"details": "An issue was discovered in drivers/accessibility/speakup/spk_ttyio.c in the Linux kernel through 5.9.9. Local attackers on systems with the speakup driver could cause a local denial of service attack, aka CID-d41227544427. This occurs because of an invalid free when the line discipline is used more than once.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-28941.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-28941.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.9.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-28941.json"
			}
		}
	],
	"references": [
		{
			"type": "FIX",
			"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=d4122754442799187d5d537a9c039a49a67e57f1"
		},
		{
			"type": "FIX",
			"url": "https://git.kernel.org/pub/scm/linux/kernel/git/gregkh/tty.git/commit/?h=tty-linus&id=d4122754442799187d5d537a9c039a49a67e57f1"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2020/11/19/5"
		},
		{
			"type": "WEB",
			"url": "https://lists.debian.org/debian-lts-announce/2020/12/msg00015.html"
		},
		{
			"type": "WEB",
			"url": "https://www.openwall.com/lists/oss-security/2020/11/19/3"
		},
		{
			"type": "FIX",
			"url": "https://github.com/torvalds/linux/commit/d4122754442799187d5d537a9c039a49a67e57f1"
		},
		{
			"type": "WEB",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/TITJQPYDWZ4NB2ONJWUXW75KSQIPF35T/"
		},
		{
			"type": "WEB",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ZF4OGZPKTAJJXWHPIFP3LHEWWEMR5LPT/"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2020-28941"
		}
	]
}
